DQM/L1TMonitor/python/L1TStage2uGTTiming_cfi.py

```diff
@@ -20,7 +20,9 @@
         "L1_SingleJet180",
         "L1_ETMHF130",
         "L1_HTT360er",
-        "L1_ETT2000"
+        "L1_ETT2000",
+        "L1_AXO_Nominal",
```

Comment:
what about the other years before 2024?
Will we be flooded by more warning message? see https://github.com/cms-sw/cmssw/issues/43488

Reply:
These seeds have been added in 2024 Menu. I tested with 2023 Runs and I get the warning that the Algo not found in the Trigger Menu

Reply:
precisely. What does the L1T DPG plan to do about these additional warnings?

Reply:
Hi @mmusich , I think the easiest way for this would be if we had year based eras in Run 3. In the meanwhile I think modifying the code to remove this warning message is ok. Atul said that the plots already omit the non-existing seeds, so there might be no reason to keep this warning in place. We'll have a look and provide a fix for these messages soon.

Reply:
Hi @eyigitba 

> I think the easiest way for this would be if we had year based eras in Run 3. 

now we do, see please https://github.com/cms-sw/cmssw/pull/43761.

Reply:
@mmusich , thanks! I wasn't aware of this. We should definitely add this to the config, but in the meanwhile we can still suppress the warning message if we added one of these seeds in the middle of a year.

Reply:
@jaisatul , @eyigitba ping.